msal/application.py

```diff
@@ -1921,7 +1921,7 @@ def __init__(self, client_id, client_credential=None, **kwargs):
 
         .. note::
 
-            You may set enable_broker_on_windows and/or enable_broker_on_mac to True.
+            You may set enable_broker_on_windows and/or enable_broker_on_mac and/or enable_broker_on_linux to True.
```

Comment:
**Question:** Does the Linux broker require yet another new redirect_uri being registered? If so, please also update the documentation [here](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/766/files#diff-4d32d284fa6acf43894719eff11cb366e5969e0b61593bfd00c68a4cf9c441caR1943-R1954).